t/rr.t

```diff
@@ -103,8 +103,8 @@ subtest 'DNSKEY' => sub {
             isa_ok( $rr, 'Zonemaster::LDNS::RR::DNSKEY' );
             ok( $rr->flags == 256 or $rr->flags == 257 );
             is( $rr->protocol,  3 );
-            # Alg 8 will replace 5. Now (December 2017) both are used.
-            ok( $rr->algorithm == 5 or $rr->algorithm == 8 );
+            # Alg 8 has replaced 5. Now (February 2022) only alg 8 is used.
+            ok( $rr->algorithm == 8 );
```

Comment:
Is this really related to the PR description? I think this should be in a separate PR with a relevant description.

Reply:
Yeah, it's under the Changes section:

> And I've included @pnax's fix for the broken tests.

Reply:
Preferably it should be in a separate PR, and "@pnax's fix for the broken tests" is not much of documentation.